kite-tools-parent/kite-tools/src/main/java/org/kitesdk/cli/commands/JSONSchemaCommand.java

```diff
@@ -65,6 +65,15 @@ public JSONSchemaCommand(Logger console) {
       description="Minimize schema file size by eliminating white space")
   boolean minimize=false;
 
+  @edu.umd.cs.findbugs.annotations.SuppressWarnings(
+      value="UWF_NULL_FIELD",
+      justification = "Field set by JCommander")
+  @Parameter(names={"-n", "--num-records"},
+             description="Number of records to derive schema from",
+             arity=1)
```

Comment:
Yeah, I like the default. I think that might not be JCommander though, I think it is my help code: https://github.com/kite-sdk/kite/blob/master/kite-minicluster/src/main/java/org/kitesdk/minicluster/cli/Help.java#L123

We should probably fix that eventually, but arity=1 is a good work-around for now.